### PR TITLE
Don't render boolean attributes that are false.

### PR DESCRIPTION
**Pull request by mbrandonw**
Don't know if I solved this the right way, but it at least fixed my test!

### CODE REVIEW

File: Tests/HtmlTests/HtmlTests.swift [hunk 1]
```diff
@@ -206,4 +206,12 @@ class HTMLTests: XCTestCase {
 
     assertSnapshot(matching: doc)
   }
+
+  func testBooleanAttributes() {
+    let uncheckedInput = input([id("not-checked"), checked(false)])
+    XCTAssertEqual("<input id=\"not-checked\">", render(uncheckedInput))
```
Comment (mbrandonw) [inline Tests/HtmlTests/HtmlTests.swift]: this test fails without the change

File: Sources/Html/Html.swift [hunk 2]
```diff
@@ -85,6 +85,13 @@ public func comment<T>(_ content: StaticString) -> ChildOf<T> {
 
 extension Value {
   public func render(with key: String) -> EncodedString? {
+
+    if let boolValue = self as? Bool {
```
Comment (stephencelis) [inline Sources/Html/Html.swift]: Should this be in `extension Bool: Value`?

File: Sources/HttpPipelineTestSupport/HttpPipelineTestSupport.swift [hunk 3]
```diff
@@ -5,6 +5,12 @@ import Prelude
 import SnapshotTesting
 import XCTest
 
+extension Application {
+  var isOther: Bool {
+    return !["javascript", "json", "xml", "x-www-form-url-encoded"].contains(self.rawValue)
```
Comment (stephencelis) [inline Sources/HttpPipelineTestSupport/HttpPipelineTestSupport.swift]: This is kinda a hack just for this file. Wonder if we can tackle the logic that depends on it in a different way.

**Comment by stephencelis** [comment]
Merging this because it's late and I tiredly spent 30 min trying to figure out why the pricing page wasn't changing with boolean values. Duh.

**Pull request merged**